packages/store-api/src/typeDefs/review.graphql

```diff
@@ -0,0 +1,9 @@
+type StoreReviewRating {
+  ratingValue: Float!
+  bestRating: Float!
+}
+
+type StoreReview {
```

Comment:
Should we aim to provide a very simple base type here or try to cover more scenarios? For example, I would include a `reviewText` or `reviewComment` field here to support this use case... but should we? 🤔  What do you think?

Reply:
I think we should include those, but in a future PR where we actually implement reviews. Let's try to keep this PR to a minimum